x-pack/test/cases_api_integration/security_and_spaces/tests/common/index_common_trial/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  createSpacesAndUsers,
+  deleteSpacesAndUsers,
+  activateUserProfiles,
+} from '../../../../common/lib/authentication';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ loadTestFile, getService }: FtrProviderContext): void => {
+  describe('Common', function () {
```

Comment:
This suite title is already used by the suite in `common/index.ts` which is nested under this suite. I'd suggest to replicate the suite suite from `trial/index.ts` (which is the level we're actually duplicating here) and make it something like
 ```suggestion
  describe('cases security and spaces enabled: trial', function () {
```